features/dd-sdk-android-ndk/README.md

```diff
@@ -0,0 +1,48 @@
+# Datadog Android Native Crash Collection
```

Comment:
We should also move this to the docs repo.

Reply:
Fair point, I created a task for that.